package.json

```diff
@@ -33,9 +34,11 @@
   },
   "devDependencies": {
     "dockerode": "^2.4.3",
-    "eslint": "^2.13.1",
-    "eslint-config-loopback": "^4.0.0",
-    "loopback-datasource-juggler": "^3.0.0",
+    "eslint": "^5.1.0",
+    "eslint-config-loopback": "^10.0.0",
```

Comment:
Update of eslint-config-loopback causes a lot of reformatting (adds a lot of insignificant white-space changes), which makes it difficult to see what changes are relevant to the introduction of the shared tests.

Please open a new pull request to update eslint config to the latest version & fix linting problems. Once that PR is landed, rebase this PR on top of the new master so that it contains only changes relevant to the introduction of the shared tests.